plugins/openselected.lua

```diff
@@ -62,5 +90,5 @@ contextmenu:register("core.docview", {
 })
 
 
-keymap.add { ["ctrl+alt+o"] = "open-selected:open-selected" }
+keymap.add { ["ctrl+shift+o"] = "open-selected:open-selected" }
```

Comment:
What's the reason for the keymap change? This will conflict with a default Lite XL shortcut.